srml/support/procedural/src/storage/transformation.rs

```diff
@@ -239,9 +239,15 @@ fn decl_store_extra_genesis(
 				has_trait_field = true;
 			}
 
-			serde_complete_bound.insert(type_infos.value_type);
+			let value_type = &type_infos.value_type;
+			serde_complete_bound.push(quote!( #value_type ));
 			if let DeclStorageTypeInfosKind::Map { key_type, .. } = type_infos.kind {
```

Comment:
also do doublemap. and maybe add more test:
if I remove Foo test and replace by `pub Foo get(foo) config(): double_map u32, blake2_256(T::BlockNumber) => Option<u32>;`
but if Foo is there is works, so maybe add more associated type

Reply:
What?

Reply:
Added a test for double map.

Reply:
Yes but the test you added already works without the add of the bound of the second key as the bound is already explicited by the value.

Reply:
Modified again, the new double map test failed on master.

Reply:
oh yes way simpler :-)